documentation/book/con-scram-sha-authentication.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-user-operator.adoc
+// assembly-configuring-kafka-listeners.adoc
+
+[id='con-scram-sha-authentication-{context}']
+= SCRAM-SHA authentication
+
+{ProductName} can configure Kafka to use SASL SCRAM-SHA to provide authentication on both unencrypted and TLS-encrypted _client connections_.
```

Comment:
Minor edits and reorg of content:

SCRAM (Salted Challenge Response Authentication Mechanism) is an authentication protocol that can establish mutual authentication using passwords. {ProductName} can configure Kafka to use SASL SCRMA-SHA-512 to provide authentication on both unencrypted and TLS-encrypted client connections. TLS authentication is always used internally between Kafka brokers and Zookeeper nodes. When used with a TLS client connection, the TLS protocol provides encryption, but is not used for authentication.

The following properties of SCRAM make it safe to use SCRAM-SHA even on unencrypted connections: